content/go/concepts/math-functions/terms/exp/exp.md

```diff
@@ -0,0 +1,74 @@
+---
+Title: 'Exp()'
+Description: 'Returns the value of e raised to the power of the parameter x.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Numbers'
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Methods'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/computer-science'
+---
+
+The **`Exp()`** function returns the value of _e^x_ (_e_ to the power of _x_), where _x_ is the parameter.
+
+## Syntax
+
+```pseudo
+import "math"
+
+value := math.Exp(number)
+```
+
+Where `value` is the value of _e_ raised to the power of `number`, and where `number` is a variable of type float64.
+
+## Special Cases
+
+- `Exp(+Inf)` returns `+Inf`
+- `Exp(NaN)` returns `NaN` as well (Not a Number)
+
+## Example
+
+The following code calculates the value of _e_ to the power of 3, and prints the resulting value to the console.
+
+```go
+package main
+
+import (
+  "fmt"
+  "math"
+)
+
+func main() {
+  result := math.Exp(3)
+  fmt.Printf("%.1f\n", result)
+}
+```
+
+The output will be:
+
+```shell
+20.1
+```
+
+## Codebyte Example
```

Comment:
Can we have a codebyte that's significantly different from the prior example? If that isn't doable, maybe we can ditch the example and just leave the codebyte.